policy_test.go

```diff
@@ -367,6 +384,13 @@ func TestBucket_GetMaxSelection(t *testing.T) {
 	root, err = newRoot(buckets...)
 	require.NoError(t, err)
 
+	t.Run("invalid filter", func(t *testing.T) {
+		ss = []Select{{Key: "Country", Count: 1}}
+		fs = []Filter{{Key: "InvalidFilter", F: FilterIn("Germany", "Spain")}}
+		r = root.GetMaxSelection(SFGroup{Selectors: ss, Filters: fs})
+		require.Nil(t, r)
+	})
+
```

Comment:
Can we add one more test to _match_ unique attribute? For example `City` attribute missed in couple of nodes. Therefore this test will fail in current master branch, but won't fail with the fix:
```go
	t.Run("match uniq attribute", func(t *testing.T) {
		ss = []Select{{Key: NodesBucket, Count: 2}}
		fs = []Filter{{Key: "City", F: FilterEQ("Madrid")}}
		r = root.GetMaxSelection(SFGroup{Selectors: ss, Filters: fs})
		require.NotNil(t, r)

		ss = []Select{{Key: NodesBucket, Count: 3}}
		r = root.GetMaxSelection(SFGroup{Selectors: ss, Filters: fs})
		require.Nil(t, r)
	})
```

Reply:
Done!